.env.dist

```diff
@@ -1,70 +1,70 @@
 # App configuration
+#############################
+## Required variables
+#### This value is used to set the sender's address in emails
 SENDER_ADDRESS=noemail@example.com
-ALLOW_FRONT_REDIRECT_PATTERN='/^http[s]*:\/\/(example.com|example2.com)/'
-WEB_CLIENT_URL=
-#Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
-SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
-OAUTH_ALLOWED_DOMAINS=example.com,gmail.com
 
-## Replace with your UID(echo $UID)
-PARENT_USER_ID=1001
-# Docker configuration!
-SERVER_HTTP_PORT=80
+#### Access to admin panel
+##### Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
+SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
```

Comment:
@mg-4xxi @beshkenadze There is a suggestion to generate this variable at startup. Using Docker ARG